src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -93,6 +93,10 @@
 #include "jvmci/jvmciJavaClasses.hpp"
 #endif
 
+#ifdef AIX
+#include <unistd.h>
+#endif
```

Comment:
This should be part of utilities/globalDefinitions.hpp. Most of the standard headers are. So maybe just include that instead, then you don't need the ifdef AIX.

Reply:
Made this change. Thanks!